src/lib.rs

```diff
@@ -43,6 +44,9 @@ pub static SVM_HOME: Lazy<PathBuf> = Lazy::new(|| {
 /// The timeout to use for requests to the source
 const REQUEST_TIMEOUT: Duration = Duration::from_secs(60);
 
+/// Version beyond which solc binaries are not fully static, hence need to be patched for NixOS.
+static NIXOS_PATCH_REQ: Lazy<VersionReq> = Lazy::new(|| VersionReq::parse(">=0.8.0").unwrap());
```

Comment:
This fails with solc 0.7.6, which seems to also be dynamically linked. Maybe we should patch every file in NixOS? statically linked ones should be noop